torchgen/gen.py

```diff
@@ -434,7 +437,19 @@ def generate_static_dispatch_fallback_call(
 def static_dispatch(
```

Comment:
The function overall here looks a bit suspect. You are assuming ambient bindings from the sig context at the call site; why isn't that sig passed as an argument here?